### PR TITLE
Getting started round two

### PR DESCRIPTION
**Pull request by GartnerInsights**
Fixed errors, added a missing link and basic cleanup

### CODE REVIEW

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/1a57b5c2296f95de52356ff8de290fbfb90345dc/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/a4113ab4eaa3e2a3f2b24b154e1525bdd7bbf5db/index.html)

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ A dialog box is displayed with the name of the project populating the text field
 
 ![Project folder](../../../images/learning_content/toolkit_basics_guides/advanced_config/14_project_folder_name.png)
 
-Toolkit can work in either a Distributed Setup, where the pipeline configuration is uploaded to Shotgun and cached locally for each user, or a Centralized Setup, where users access a single configuration in a shared location on disk. For this exercise we will use a Centralized Setup. You can [learn more about Distributed Setups here](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
+Toolkit can work in either a Distributed Setup, where the pipeline configuration is uploaded to Shotgun and cached locally for each user or a Centralized Setup, where users access a single configuration in a shared location on disk. For this exercise we will use a Centralized Setup. You can [learn more about Distributed Setups here](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Sorry, one more note!

I'd like to keep this comma, otherwise it's unclear where the clause that starts with "where the pipeline configuration.." ends.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ lang: en
 
 # Getting started with configurations
 
-After completing this guide, you will have the knowledge fundamental to creating an advanced setup for your Toolkit project, setting you up customize your pipeline configuration.
+After completing this guide, you will have the knowledge fundamental to extending and customizing a configuration for your project’s pipeline.
```
[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: It sets you up to extend it for the next guide... I agree we don't extend it here, but the process we are doing is what's needed for the next guide.

[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: I don't know where you were going with the changes you made to this. The original one I had was... I didn't write setting you up...

[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Anyway, I'll change this...

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ To use this guide and create a customizable configuration for your project, the
 
 ## About the Advanced Project Setup Wizard
 
-The Advanced Project Setup Wizard in Shotgun Desktop generates a pipeline configuration based on our Default Configuration that provides a solid base to build on, adding key software integrations, customizable settings, apps and UI elements that support the pipeline process. It creates a configuration you can edit and extend to meet your project’s pipeline needs.
+The Advanced Project Setup Wizard in Shotgun Desktop generates a pipeline configuration based on a Default Configuration. The Default Configuration provides a solid base to build on complete with customizable settings, apps, and UI elements that support the pipeline process. It creates a configuration you can edit and extend to meet your project’s pipeline needs.
```
[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: OK I'll make those changes.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: We're getting ride of advanced setup

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b555d1070b5d9dfa7b7430dcf80615091254df8e/index.html)

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md (hunk 1 above)
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: OK, but the "or" is what created the separation... it's not correct english... I think it's clear without the comma.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 4]
```diff
@@ -67,13 +67,13 @@ In this guide, you will use the Wizard in Shotgun Desktop to generate a pipeline
...
 
-**Step 3:** Navigate to the project where you wish to run the Advanced Project Setup Wizard by selecting its thumbnail image.
+**Step 3:** Navigate to the project where you wish to run the Wizard by selecting its thumbnail image.
 
 {% include info title="Tip" content="If a new project was created while Desktop was open, the **Projects** window may need to be refreshed to see the new projects. Select your profile **avatar** at the bottom right of the desktop and select Refresh Projects." %}
 
 ![Refresh project](../../../images/learning_content/toolkit_basics_guides/advanced_config/5_refresh_projects_list.png)
 
-## Creating the Advanced Configuration
+## Creating the Default Configuration
```
[the remark this answers is not in the document]
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: OK, that's what I had in the edit doc

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/80a6fb970013e3bf5bd9134d82491f0faccde2ef/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b18a6ba1585d19f875993d2f9a4a60f2ee817cbd/index.html)

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ lang: en
 
 # Getting started with configurations
 
-After completing this guide, you will have the knowledge fundamental to extending and customizing a configuration for your project’s pipeline.
+After completing this guide, you will have the knowledge fundamental to adding your project information to a Default Configuration, associating that configuration with your project, and preparing your pipeline configuration to be customized.
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Again, we're not creating a Default Config here, just creating a config for our project based on the Default Config. How about this: 

After completing this guide, you will have the knowledge fundamental to creating an editable configuration for your project based on the Default Configuration.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 6]
```diff
@@ -155,13 +155,13 @@ And now the real fun begins, learning all the things you can do with the Configu
...
+Shotgun Toolkit provides many convenient ways to edit, clone, or take over a configuration. Extending existing configurations will save time and allow you access to all of the cool stuff that others within your network have created. You can take advantage of the vast Shotgun community (https://groups.google.com/a/shotgunsoftware.com/forum/?fromgroups#!forum/shotgun-dev) that may have the exact configuration you need. The Shotgun community is a sharing community, so be kind, say thank you, and recognize the person who created the configuration that helped you get the job done. Oh, and don’t forget to give back, it’s how we help our fellow Shotgun gurus and what makes it so special to be a part of this community!
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: not sure why this change was made. looks like we lost the markdown for the link to the shotgun-dev list?

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 7]
```diff
@@ -155,13 +155,13 @@ And now the real fun begins, learning all the things you can do with the Configu
...
+From inside any project configuration, the `tank` command lets you run administrative commands from a terminal. Each project has its own dedicated `tank` command. The `tank setup_project` command's functionality is analogous to the Advanced Setup Wizard's: it creates an editable configuration on disk for your project based either on an existing project's configuration or the Default Configuration. You can learn more about running [`tank setup_project` here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#setup_project), and more about the  [`tank` command here](https://support.shotgunsoftware.com/hc/en-us/articles/219033178-Administering-Toolkit#Using%20the%20tank%20command).
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: I'm not sure why you made this change  – can you clarify?

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 8]
```diff
@@ -178,10 +178,10 @@ To learn about cloning your configuration, as well as other basics of configurat
...
+This guide walks through creating a **centralized configuration**: a single copy of a pipeline configuration that is located in a shared network location and accessed by all users. But, if your workforce is distributed across locations and networks, it may not be feasible to share a single copy of a pipeline configuration. Toolkit allows for this scenario by providing a **distributed configuration** option where a pipeline configuration is uploaded to Shotgun and each time a user accesses a project the configuration is downloaded and cached locally. You can learn more about distributed configurations in our [Core API developer documentation](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Same goes for this one. Not sure why you made this change.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md [hunk 9]
```diff
@@ -178,10 +178,10 @@ To learn about cloning your configuration, as well as other basics of configurat
...
 
 ## Working with more than one root folder
 
-Ideally your facility would want to be optimized for specific tasks. You can work with more than one root folder to optimize things like video playback for dailies on one server and interactive processing on another. Toolkit allows you to work with more than one storage root, in order to facilitate workflows such as these. You can [learn more about multi-root configurations here]((https://support.shotgunsoftware.com/hc/en-us/articles/219040478-How-do-I-convert-my-project-from-a-single-root-to-a-multi-root-configuration-)). 
+Ideally your facility would want to be optimized for specific tasks. You can work with more than one root folder to optimize things such as video playback for dailies on one server and interactive processing on another. Toolkit allows you to work with more than one storage root in order to facilitate workflows such as these. Check out how to convert from a [single root to a multi-root configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219040478-How-do-I-convert-my-project-from-a-single-root-to-a-multi-root-configuration-) and learn more about a distributed configuration.
```
Comment (tannazie) [inline docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Not sure why you made this change, and there is an error – this has nothing to do with distributed configurations.

**Comment by GartnerInsights** [comment]
Did you read the comments in the PR? it's all in there.

The last edit was a mistake and meant for the paragraph prior. I'll fix
that.

*Michelle F. Gartner*
WORDS - Creating thought provoking experiences that invoke curiosity and
invite exploration.

Office: 530-546-6007
Mobile: 530-240-5957

Website: http://gartnerinsights.com/
Linkedin: https://linkedin/in/michellegartner
<https://www.linkedin.com/in/michellegartner>
Blog: http://magicdoormarketing.blogspot.com

May our journey together be brilliant and effective.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md (hunk 5 above)
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: I'm a bit confused. I thought the Wizard allows the user to add their project information to a default config etc... exactly what I wrote. I thought that sounded good and is exactly what is being done. There's a Default configuration that the wizard is editing to add the users project information to. I don't say you're creating a default configuration anywhere.

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md (hunk 6 above)
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: Any link that takes the user away from our content is supposed to be explicit

File: docs/en/toolkit/toolkit_basics_guides/advanced_config.md (hunk 7 above)
Comment (GartnerInsights) [reply docs/en/toolkit/toolkit_basics_guides/advanced_config.md]: I noticed you added the cloud distribution section and was wondering if we want to talk a little about the benefits of a distributed configuration? Like the users always have the most up to date config since it's cached every time... I tried to imply this. Let me know what you think.
This was in the PR comment ... For the links in the advanced topics section... The community link is an external link, so it's visible in the text. The style guide states to use doc name or a term reference in the support guide as a link. See Style Guide... https://wiki.autodesk.com/pages/viewpage.action?spaceKey=SHOT&title=Style+Guide#StyleGuide-Linkingtothesupportsite  I left the tank links in, but moved the link to only the words "here"...
These should change too and remove the word here making the link something else... I didn't know what to change it to and wanted to leave that one to you.

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/abac4a0b736c2c089f08f73bbb1671cb9f7e0f15/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/dd22faa6dc9a1ac66a3140fdc034b8bed1b5b4df/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/b9be65686b2c6f54678ab4f1741a5b0f077c5c82/index.html)

**Comment by manneohrstrom** [comment]
[Documentation Preview](http://doctest3.s3-website.eu-west-2.amazonaws.com/tk-doc-generator/bb95482a866c030f56f7d11b7381d25b67f0b1e4/index.html)
